js/settings-admin.js

```diff
@@ -19,24 +19,38 @@
  */
 
 $(document).ready(function(){
-    $('#password-policy').on('click','#save-pass-length',
+    $('#security-brute-force-protection').on('click','#save-bfp-settings',
         function() {
-            OC.msg.startSaving('#of-msg');
+            OC.msg.startSaving('#security-save-bfp-settings-message');
+            var thresholdTime = $('#security-bfp-threshold-time').val();
+            var failTolerance = $('#security-bfp-fail-tolerance').val();
+            var banPeriod = $('#security-bfp-ban-period').val();
+            if($.isNumeric(thresholdTime) && thresholdTime>0 &&
+                $.isNumeric(failTolerance) && failTolerance>0 &&
+                $.isNumeric(banPeriod) && banPeriod>0) {
+                OC.AppConfig.setValue('security', 'brute_force_protection_time_threshold', thresholdTime);
+                OC.AppConfig.setValue('security', 'brute_force_protection_fail_tolerance', failTolerance);
+                OC.AppConfig.setValue('security', 'brute_force_protection_ban_period', banPeriod);
+                OC.msg.finishedSuccess('#security-save-bfp-settings-message', OC.L10N.translate('Security', 'Preferences are saved'));

```

Comment:
please use t() instead of OC.L10N.translate - only using t() the extraction tool can extract strings - THX